app/models/concerns/wp_term.rb

```diff
@@ -0,0 +1,29 @@
+module WpTerm
+  extend ActiveSupport::Concern
+
+  def update_term(json)
+    self.class.mappable_wordpress_attributes.each do |wp_attribute|
+      send("#{wp_attribute}=", json[wp_attribute])
+    end
+
+    self.wp_id        = json['ID']
+    if json['parent']
+      self.class.where(wp_id: json['parent']['ID']).first_or_create.update_wp_cache(json['parent'])
```

Comment:
Line is too long. [99/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.